src/ewasm/codegen/imports.rs

```diff
@@ -0,0 +1,166 @@
+extern crate inkwell;
+
+use crate::ewasm::codegen::Codegen;
+use inkwell::attributes::{Attribute, AttributeLoc};
+use inkwell::module::Linkage;
+use inkwell::types::{BasicType, FunctionType};
+use inkwell::values::BasicValue;
+use inkwell::AddressSpace;
+
+// For now this will just set up what will often likely need, i.e. getCaller and revert
+impl<'a, 'ctx> Codegen<'a, 'ctx> {
+    pub fn ether_imports(&self) {
+        // self.import_get_caller(); // Using a dummy getCaller while we are testing TODO remove for release
+        self.import_revert();
+        self.import_get_external_balance();
+        self.import_call();
+        self.import_get_gas();
+    }
+
+    #[allow(dead_code)]
+    fn import_get_caller(&self) {
```

Comment:
Look at using proper function documentation comments for rust doc, and possibly building documentation?